skype/skype-ps/skype/New-CsOnlineApplicationInstance.md

```diff
@@ -0,0 +1,94 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+applicable: Skype for Business Online
+title: New-CsOnlineApplicationInstance
+schema: 2.0.0
+author: kenwith
+ms.author: kenwith
+ms.reviewer:
+---
+
+# New-CsOnlineApplicationInstance
+
+## SYNOPSIS
+Creates an application instance in AAD. The application instance will be sync'ed into LyncAD and BVD after it's created.
```

Comment:
Vinod to confirm if we should reveal the sync tech details.

Reply:
@elvin-ms @zhengni-ms  
Since this is public documentation, we shouldn't include terms like LyncAD or BVD, these look like implementation details.